new_platforms/vscode-extension/SetupDevice.md

```diff
@@ -0,0 +1,97 @@
+# Configure Azure IoT Edge Device
```

Comment:
nit: The filename capitalization/grammar is odd, since "setup" is a noun ("set up" is a verb).  So should probably be either "DeviceSetup.md" or "SetUpDevice.md" depending on intended meaning.